info.plist

```diff
@@ -1202,6 +1202,11 @@
 			<integer>770</integer>
 		</dict>
 	</dict>
+    <key>variables</key>
+    <dict>
+        <key>PATH</key>
+        <string>/usr/local/bin:/usr/bin:/Library/Frameworks/Python.framework/Versions/2.7/bin</string>
+    </dict>
```

Comment:
Required otherwise the workflow fails running with error `[11:43:38.295] ERROR: gist[[Script Filter](alfredpreferences:workflows%3Eworkflow%3Euser.workflow.EA39ED05-ED6A-4A48-8AFB-E3EDE850A46B%3E78B3D890-4494-4AA6-A8C0-476140DBEF0C)] Code 127: env: python: No such file or directory`.

Reason is that Alfred doesn't inherit env variables and it doesn't source `~/.zshrc` & co.

Reply:
Homebrew doesn't have Python 2 anymore (since its support stopped). This is the reason why I omitted it.

I'm also on Apple Silicon.

Reply:
> Homebrew doesn't have Python 2 anymore (since its support stopped).

I am aware, I am a Homebrew maintainer. But Python 2 can be brought back to the Homebrew location by following [the official knowledge base article](https://www.alfredapp.com/help/kb/python-2-monterey/) and Alfred will pick it up. It’s the method used for the [Updated Third-Party Python 2 Workflows repo](https://github.com/alfredapp/updated-third-party-python2-workflows). It works on Apple Silicon and is in active use.

By adding `/opt/homebrew/bin`, you also fix the Workflow for the people who followed that guide, of which there are many (every Workflow [on the table](https://github.com/alfredapp/updated-third-party-python2-workflows#workflows) was requested by someone, a few by multiple people).